test/parallel/test-runner-bail.js

```diff
@@ -0,0 +1,174 @@
+'use strict';
```

Comment:
There seems to be a lot of overlap with the changes to `test/parallel/test-runner-run.mjs`. That other file is getting pretty large, and `--test` already goes through `run()`, so I'd say eliminate any duplication that isn't needed.

Reply:
removed duplicated tests